chapter_05_aggregate.asciidoc

```diff
@@ -1,6 +1,74 @@
 [[chapter_05_aggregate]]
 == Aggregates and consistency boundaries
 
+What's the point of a domain model anyway? What's the fundamental problem
+we're trying to addresss?
+
+Couldn't we just run everything in a spreadsheet? Many of our users would be
+delighted by that. Business users _like_ spreadsheets because they're simple and
+familiar but offer .
+
+In fact, an enormous number of business processes do operate by manually sending
+spreadsheets back and forward over e-mail. This "csv over smtp" architecture has
+low initial complexity but tends not to scale very well because it's difficult
+to apply logic.
+
+// TODO: better examples.
```

Comment:
what happens when two different people try to update the spreadsheet at the same time?